test/cmd/newapp.sh

```diff
@@ -9,10 +9,18 @@ source "${OS_ROOT}/hack/util.sh"
 source "${OS_ROOT}/hack/cmd_util.sh"
 os::log::install_errexit
 
+# Cleanup cluster resources created by this test
+(
+  set +e
+  oc delete all,templates --all
+  exit 0
+) &>/dev/null
```

Comment:
So we can rerun them as an end user - the original point of all the
individual scripts is that you can re run them over and over.

On Tue, Feb 2, 2016 at 3:19 AM, Rodolfo Carvalho <notifications@github.com>
wrote:

> In test/cmd/newapp.sh
> <https://github.com/openshift/origin/pull/6670#discussion_r51536701>:
>
> > @@ -9,10 +9,18 @@ source "${OS_ROOT}/hack/util.sh"
> >  source "${OS_ROOT}/hack/cmd_util.sh"
> >  os::log::install_errexit
> >
> > +# Cleanup cluster resources created by this test
> > +(
> > +  set +e
> > +  oc delete all,templates --all
> > +  exit 0
> > +) &>/dev/null
>
> Is this supposed to be in the final patch?
> Why do we delete resources prior to running the tests?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6670/files#r51536701>.
>